Makefile

```diff
@@ -33,6 +33,53 @@ $(PROG): $(OBJ:%=$(BUILDDIR)/%)
 $(BUILDDIR)/%.o: %.c | builddir
 	$(CC) -MMD -c $(CFLAGS) -o $@ $<
 
+fip:
+	if [ -d "$(BUILDDIR)/fip" ]; then rm -r "$(BUILDDIR)/fip"; fi
+ifdef FIP
+	cp -r "$(FIP)/" "$(BUILDDIR)/fip"
+else
+	curl -L https://github.com/BayLibre/u-boot/releases/download/v2017.11-libretech-cc/libretech-cc_fip_20180418.tar.gz | tar -xvz -C $(BUILDDIR)
+endif
+
+$(BUILDDIR)/fip/gxl/bl2_acs.bin: fip
```

Comment:
It would be better to not download everything each time make image is called. Maybe make that target depending of the tarball and add a rule to clean this tarball (e.g. make image-clean)